docs/reference/src/main/sitemap/sitemap.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>http://docs.jboss.org/weld/reference/latest-master/en-US/html/part-1.html</loc>
```

Comment:
We should probably make the url base, i.e. `http://docs.jboss.org/weld/reference/latest-master/en-US/html`, configurable. A property in docs/reference/pom.xml should work. What do you think?